tests/src/test/java/org/sonarqube/tests/qualityGate/QualityGateTest.java

```diff
@@ -257,23 +260,37 @@ public void ad_hoc_build_break_strategy() throws IOException {
 
   @Test
   public void does_not_fail_when_condition_is_on_removed_metric() throws Exception {
+
+    // create project
+    Project project = tester.projects().generate(null);
```

Comment:
Organizations are not enabled in `Category1`, so we have to use the default organization. :(